### DISCUSSION TITLE
AuthJS/NextAuth - Able to Use Existing Users

### DISCUSSION THREAD

**Discussion by Apezdr** [notifo-io/notifo#226] [Q&A]
It may be possible and I haven't looked deep enough though I am curious if I can use my existing user collection in MongoDB managed by the [AuthJS/NextAuth](https://authjs.dev/).

Currently it creates a collection for users, sessions, etc. having the notification system hook up to this directly would make using this a MUCH simpler solution than home brewing a notification system. For context this is for a NextJS 14 application.

**Comment by SebastianStehle** [comment]
No right now,

Notifo has to resolve the user during notifications to resolve language information and so on. So it makes sense to keep them local.

BUT you can use your own ID in notifo, therefore you can just use the upsert request to get the token, e.g. on each login.
